### DISCUSSION TITLE
How to enable Module's attribute drag and drop?

### DISCUSSION THREAD

**Discussion by UmerTariqKhan** [mixcore/mix.core#445] [Q&A] [answered]
How to enable Module's attribute drag and drop?
![image_2021_08_01T19_56_11_096Z](https://user-images.githubusercontent.com/36357077/127783746-5e112474-17cc-4381-9115-ebe6c729ca00.png)

**Comment by Smilefounder** [comment, marked as answer]
Hi @UmerTariqKhan , Kindly change the filter on the sidebar with the condition below.:
- Order by: Priority
- Order direction: Asc 

By the way, You can pull the latest source code from Master.
